### ISSUE TITLE
Update yellow smiley

### ISSUE THREAD

**Issue by hedgefield** [Yoast/yoast-components#702]
The smiley for the analysis has a lighter color than the bullet. This is because the smiley's yellow has better contrast for peole with visual impairments. However it does not have enough contrast with the background to work without the smiley face on it (so if it were used for the bullets), so we have to change the color.

This can be done by changing the following line in `wordpress-seo/js/src/components/contentAnalysis/mapResults.js` from:

`icon = { icon: "seo-score-ok", color: colors.$color_yellow_score };`

to:

`icon = { icon: "seo-score-ok", color: colors.$color_ok };`

Secondly, regardless of the color, the smiley face does not have enough contrast with it as is. We wanted to make the face black, but it turns out that the way we load SVGs only supports one 'path', and by making the face black it adds three more paths, which cannot be used. @afercia did try to work around this with some CSS, but it would be much nicer if the SVG file supported multiple paths per icon.

**Comment by afercia** [comment]
Was playing a bit with CSS and got something, but it's really a hack 😬 Not sure if there are other ways to change the smiley other than refactoring the SVG...
```
	.yoast-svg-icon-seo-score-ok {
		background: black;
		border-radius: 50%;
		box-shadow: inset 0px 0px 0px 2px #f5c819;
	}
```

**Comment by afercia** [comment]
I’ve found a way to render SVG icons with multiple shapes, so I’m going to use the first version of the icon @hedgefield prepared instead of dirty CSS hacks. Needs a PR on yoast-components.

**Comment by hedgefield** [comment]
Great! I just posted an issue for SVG icon handling improvements, where the multi-shape thing is mentioned, if you could post your findings: https://github.com/Yoast/yoast-components/issues/705
